samples/hello_world/boards/nrf54h20pdk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,7 @@
+&cpuppr_ram3x_region {
+	status = "okay";
+};
+
+&cpuppr_vpr {
+	status = "okay";
+};
```

Comment:
Adding this should get PPR working:
```
&uart135 {
	status = "reserved";
};
```

Reply:
indeed, that fixes the problem. I see that UICR gets the pins this way.